src/Fossa/API/Types.hs

```diff
@@ -356,7 +356,8 @@ instance FromJSON IssuesSummary where
       <$> obj
         .: "revision"
       <*> obj
-        .: "targets"
+        .:? "targets"
+        .!= []
```

Comment:
[Optional] I think older versions may not, but the newer CI/Fourmolu/HLS 2.7.0.0 should accept:
```haskell
      <$> obj .: "revision"
      <*> obj .:? "targets" .!= []
```
Which I think is nicer.

Reply:
omg thank you, added a commit that fixes this across the codebase.

Reply:
Even better than Christmas!